pkg/graph/schema.graphql

```diff
@@ -1652,6 +1663,16 @@ enum TRBFormStatus {
   COMPLETED
 }
 
+"""
+Represents the status of the TRB step
+"""
+enum TRBTaskStatus {
```

Comment:
I'd prefer separating this into a `TRBFeedbackStatus` enum (with values `READY_TO_START`, `IN_REVIEW`, `EDITS_REQUESTED`, and `COMPLETED`) and a `TRBConsultStatus` enum (with values `READY_TO_START`, `IN_PROGRESS`, and `COMPLETED`). This more clearly distinguishes them, as well as showing the difference between feedback being "in review" and a consult being "in progress". The repetition is admittedly annoying, though.

Reply:
Seems it could go either way - I was wanting to combine them further and just have a couple flags in one enum that don't necessarily apply to each step.

Reply:
After discussion with Jeremy, Clay, Steven, and I, we're going to split this into separate enums.

Reply:
Looks good as of https://github.com/CMSgov/easi-app/pull/1824/commits/4d1bb554ca5fc567488ee859ca75e4aa4ebb951d.